frontend/app/src/entities/diff/checks/check.tsx

```diff
@@ -186,11 +163,25 @@ export const Check = ({ id }: tCheckProps) => {
         </div>
       </div>
 
-      {content && (
-        <div>
-          <div className="border-t-2 border-gray-100 mb-2 rounded-md" />
+      {conflicts?.value?.length && (
```

Comment:
if length = 0, it'll be visible:
<img width="782" alt="image" src="https://github.com/user-attachments/assets/978a77da-c948-4c1d-bbd4-f0bcf710d3a6" />